src/styles/scss/tools/variables.colors.scss

```diff
@@ -13,9 +13,15 @@ $color__identity-primary: hsl(
   1
 );
 
+$color__page-bg: $color__global-primary;
+
 $color__text-primary-white: $color__global-white;
 $color__text-primary-black: $color__global-black;
 $color__text-secondary-gray: $color__global-grey;
+$color__text-primary-white: #fff;
+$color__text-primary-black: #000;
+$color__text-secondary-gray: #484848;
```

Comment:
I think all are these are duplications to the ones just above aren't they?

in ddfform-59 i added global black/white/grey and changed the __text ones to use those.

Reply:
You're right, it's a merge conflict, i'll fix it.